a/apache-arrow/arrow_ubi_9.3.sh

```diff
@@ -0,0 +1,149 @@
+#!/bin/bash -e
+# -----------------------------------------------------------------------------
+#
+# Package          : arrow
+# Version          : go/v16.1.0
+
+# Source repo      : https://github.com/apache/arrow
+# Tested on        : UBI 9.3
+# Language         : C++,Go
+# Travis-Check     : True
+# Script License   : Apache License, Version 2 or later
+# Maintainer       : Puneet Sharma <Puneet.Sharma21@ibm.com>
+#
+# Disclaimer       : This script has been tested in root mode on given
+# ==========         platform using the mentioned version of the package.
+#                    It may not work as expected with newer versions of the
+#                    package and/or distribution. In such case, please
+#                    contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+PACKAGE_NAME=arrow
+PACKAGE_URL=https://github.com/apache/arrow.git
+PACKAGE_VERSION=${1:-go/v16.1.0}
+PYTHON_VER=${2:-"3.11"}
```

Comment:
We should not be having `PYTHON_VER` and python installation steps in build-script, as there is wrapper script that installs all the python versions and creates the python venv and runs this build-script.

Reply:
Ok Thanks @shubham-dayma-ibm